pkg/db/factory.go

```diff
@@ -21,14 +22,15 @@ func NewConnectionFactory(configFile string, debug bool) (ConnectionFactory, err
 	}
 
 	dbToUse := viper.GetString("db.driver")
+	executeUnverified := viper.GetBool("db.executeUnverified")
```

Comment:
We could use a kind of `sqlValidator` instance and pass this instance it the conn-factories (under the assumption, that it's stateless and can be used in parallel without issues).